dev/devicelab/lib/framework/utils.dart

```diff
@@ -145,14 +144,14 @@ void section(String title) {
   title = '╡ ••• $title ••• ╞';
   final String line = '═' * math.max((80 - title.length) ~/ 2, 2);
   String output = '$line$title$line';
-  if (output.length == 79)
-    output += '═';
+  if (output.length == 79) output += '═';
   print('\n\n$output\n');
 }
 
 Future<String> getDartVersion() async {
   // The Dart VM returns the version text to stderr.
-  final ProcessResult result = _processManager.runSync(<String>[dartBin, '--version']);
+  final ProcessResult result =
+      _processManager.runSync(<String>[dartBin, '--version']);
```

Comment:
Nit: here and below, retain the existing formatting for unrelated code

Reply:
Done.